core/src/components/button/button.ios.vars.scss

```diff
@@ -34,7 +34,7 @@ $button-ios-height:                                   3.1em !default;
 $button-ios-border-radius:                            14px !default;
 
 /// @prop - Font size of the button text
-$button-ios-font-size:                                16px !default;
+$button-ios-font-size:                                min(1rem, 48px) !default;
```

Comment:
I set the maximums to the value at 310% on `ios`. The actual value of the `font-size` at 310% is in between 46 and 47px but I thought it would be nicer to use a value 3x the normal one.

Reply:
Do you think it's worth documenting how we derived these values somewhere?

Reply:
Yeah probably. If you agree with the approach I could add comments above the values.

Reply:
yeah I think doing the 3x scaling makes sense